src/Config/CliBuilder.ts

```diff
@@ -108,15 +109,21 @@ export class CliBuilder {
       const systemConfigManager = new SystemConfigManager(args.api as string);
       const systemConfig = await systemConfigManager.read();
 
-      return Sentry.runWithAsyncContext(() => {
+      return Sentry.runWithAsyncContext(async () => {
         this.initSentry(systemConfig.sentryDsn);
         Sentry.setContext('args', args);
 
         systemConfigManager.enableBackgroundRotation((rotatedSystemConfig) => {
           this.initSentry(rotatedSystemConfig.sentryDsn);
         });
 
-        return handler(args);
+        try {
+          return await Promise.resolve(handler(args));
+        } catch (e) {
+          captureException(e);
+
+          throw e;
+        }
```

Comment:
This appears to be redundant, as you're already capturing the same exception in the specific command.

Reply:
Not always. For example, `src/Commands/RunRepeater.ts` is not fully wrapped with try/catch block, but only `run` method.

Reply:
This is precisely why we decided to explicitly capture exceptions - to prevent interception of user-initiated errors. However, now you grab them as well. This is redundant and does not align with the goal of PR

Reply:
Makes sense. Removed